src/main/drivers/max7456.c

```diff
@@ -37,43 +37,42 @@
 #include "max7456.h"
 #include "max7456_symbols.h"
 
-//on shared SPI buss we want to change clock for OSD chip and restore for other devices
+// On shared SPI bus we want to change clock for OSD chip and restore for other devices
 #ifdef MAX7456_SPI_CLK
-    #define ENABLE_MAX7456        {spiSetDivisor(MAX7456_SPI_INSTANCE, MAX7456_SPI_CLK);IOLo(max7456CsPin);}
+    #define ENABLE_MAX7456  {spiSetDivisor(MAX7456_SPI_INSTANCE, MAX7456_SPI_CLK);IOLo(max7456CsPin);}
 #else
-    #define ENABLE_MAX7456        IOLo(max7456CsPin)
+    #define ENABLE_MAX7456  IOLo(max7456CsPin)
 #endif
 
 #ifdef MAX7456_RESTORE_CLK
-    #define DISABLE_MAX7456       {IOHi(max7456CsPin);spiSetDivisor(MAX7456_SPI_INSTANCE, MAX7456_RESTORE_CLK);}
+    #define DISABLE_MAX7456 {IOHi(max7456CsPin);spiSetDivisor(MAX7456_SPI_INSTANCE, MAX7456_RESTORE_CLK);}
 #else
-    #define DISABLE_MAX7456       IOHi(max7456CsPin)
+    #define DISABLE_MAX7456 IOHi(max7456CsPin)
 #endif
 
-uint16_t max_screen_size = VIDEO_BUFFER_CHARS_PAL;
+uint16_t maxScreenSize = VIDEO_BUFFER_CHARS_PAL;
 // we write everything in SCREEN_BUFFER and then comapre
 // SCREEN_BUFFER with SHADOW_BUFFER to upgrade only changed chars
 // this solution is faster then redraw all screen
 
-static uint8_t SCREEN_BUFFER[VIDEO_BUFFER_CHARS_PAL+40]; //for faster writes we use memcpy so we need some space to don't overwrite buffer
+static uint8_t SCREEN_BUFFER[VIDEO_BUFFER_CHARS_PAL + 40]; //for faster writes we use memcpy so we need some space to don't overwrite buffer
 static uint8_t SHADOW_BUFFER[VIDEO_BUFFER_CHARS_PAL];
 
 //max chars to update in one idle
 #define MAX_CHARS2UPDATE    100
 #ifdef MAX7456_DMA_CHANNEL_TX
-volatile uint8_t dma_transaction_in_progress = 0;
-#endif
+volatile bool dmaTxInProgress = false;
+#endif // MAX7456_DMA_CHANNEL_TX
 
-static uint8_t spi_buff[MAX_CHARS2UPDATE*6];
+static uint8_t spiBuffer[MAX_CHARS2UPDATE * 6];
 
-static uint8_t  video_signal_cfg   = 0;
-static uint8_t  video_signal_reg   = VIDEO_MODE_PAL | OSD_ENABLE; //PAL by default
-static uint8_t  max7456_lock        = 0;
-static IO_t max7456CsPin            = IO_NONE;
-static bool fontIsLoading = false;
+static uint8_t videoSignalCfg = 0;
+static uint8_t videoSignalReg = VIDEO_MODE_PAL | OSD_ENABLE; //PAL by default
+static bool    max7456Lock    = false;
+static IO_t    max7456CsPin   = IO_NONE;
+static bool    fontIsLoading  = false;
 
-static uint8_t max7456_send(uint8_t add, uint8_t data)
```

Comment:
Weird, I used clang-format with the Stroustrup (closest to K&R it supports) indentation style to do this so not sure why it has used this style, it worked fine is `osd.c`.

I'll add it to the fixes in #1221.